docker/docker_image_dest.go

```diff
@@ -34,6 +34,8 @@ type dockerImageDestination struct {
 	c   *dockerClient
 	// State
 	manifestDigest digest.Digest // or "" if not yet known.
+
+	acceptUncompressedLayers bool
```

Comment:
Please move this above the `// State` section marker, this is not mutable state.